src/shared/components/workbasket-filters/workbasket-filters.component.html

```diff
@@ -25,6 +25,7 @@ <h2 class="heading-h2" aria-label="Filters">Filters</h2>
     <select class="form-control form-control-3-4 ccd-dropdown" id="wb-case-state"
             name="state" [(ngModel)]="selected.caseState" [disabled]="isCaseStatesDropdownDisabled()"
             aria-controls="search-result">
+        <option [ngValue]="null">Any</option>
```

Comment:
Minor point but is there any reason this `<option>` element is indented more than the one below?